src/app/core/components/footer/footer.component.ts

```diff
@@ -3,6 +3,6 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'vc-footer',
   templateUrl: './footer.component.html',
-  styleUrls: ['./footer.component.scss'],
+  styleUrls: ['./footer.component.scss', '../shared/shared.style.scss'],
```

Comment:
Why do you put shared styles like this? didn't check, but probably they will be duplicated during compilation.
All shared styles across whole app usually are put in app.component.scss or src/styles.scss file, splitted logically by file by file. Sometimes variables or mixins are needed for components, separate file for them also usually is included inside file (for example footer.component.scss)